README.md

```diff
@@ -27,7 +26,7 @@ on installing `git` for Linux and Mac.
 libvirt to accept TCP connections for remote installations. Ensure firewall rules allow remote connections to the host.
 
 
-## Image and LPAR requirements
+# Image and LPAR requirements
```

Comment:
How about adding details of bastion/helper node VM configuration ?

Reply:
Added.